library/java/io/envoyproxy/envoymobile/EnvoyEngine.java

```diff
@@ -1,4 +1,4 @@
-package io.envoyproxy.envoymobile;
+package io.envoyproxy.envoymobile.envoyengine;
```

Comment:
Why the package change? Not that big a deal, just curious.

Reply:
It was a bit strange to have the kotlin layer have the same package name as the envoy engine layer since they're being built separately.

Reply:
Sounds good. Should we just call it 'engine'?